drgn_tools/block.py

```diff
@@ -425,7 +425,7 @@ def dump_inflight_io(prog: drgn.Program, diskname: str = "all") -> None:
             ).value_() != disk.value_():
                 continue
             print(
-                "%-20s %-20lx %-20lx %-16s\n%-20s %-20d %-20d %-16d"
+                "%-20s 0x%-20lx 0x%-20lx %-16s\n%-20s %-20d %-20d %-16d"
```

Comment:
This feels a bit random - I don't have strong feelings about "0x" but I'm confused why you're modifying this here.

Reply:
The other two functions outputs hex address with "0x" prefix so to make it consistent, i've made this change. If this seems like a unnecessary thing, i can undo this.

Reply:
Ok, then leave it in. Thanks!